actionlist/app/reducers/action_items.js

```diff
@@ -1,12 +1,23 @@
 /* eslint arrow-body-style: ["error", "as-needed", { "requireReturnForObjectLiteral": true }] */
 import { handleActions } from 'redux-actions';
-import { TOGGLE_DRAWER, SORT_ACTION_LIST } from '../actions/types';
+import { TOGGLE_DRAWER, SORT_ACTION_LIST, SELECT_DROPDOWN_OPTION } from '../actions/types';
 import { processInstances, sortTypes } from '../lib/commons';
 
 export const defaultState = {
   dataSource: processInstances,
   drawerExpanded: false,
   optionSelected: '',
+  dropdownColors: {
+    saved: 'white',
```

Comment:
Sure, but the values of the `dropdownColors` properties don't need to be string literals. You could extract `Dropdown#OPTIONS` to `commons` as 
```es6
export const preferenceColors = {
  aqua: 'aqua',
  blue: 'blue',
// ...
};
```
And then in `Dropdown`, when you render it, you can do something like 
`<ModalDropdown [...] options={preferenceColors.Values()} >...</ModalDropdown>`
and here you would have 
```es6
dropdownColors: {
  saved: preferenceColors.white,
  // ...
},
```